api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java

```diff
@@ -74,82 +69,29 @@ public void setConnectionManager(ConnectionManager connectionManager) {
     @Override
     public MotTest getMotTestById(long id) {
 
-        logger.debug("Entry getMotTestById " + id);
-        if (motTestMap.containsKey(id)) {
-            logger.debug("Exit getMotTestById Returning Cached MotTest id " + id);
-            return motTestMap.get(id);
-        } else {
-            MotTest motTest = getMotTestCurrentById(id);
+        MotTest motTest = getMotTestCurrentById(id);
```

Comment:
A lot of the logging seemed over the top imho as there is already entered/exited logging at the service level. I have created an Aspect to replace this logging but didn't include it in the commit. I'll push it up so that it can be reviewed.